astropy/coordinates/utils.py

```diff
@@ -0,0 +1,15 @@
+def generate_docs():
+    output = []
+    try:
+        from astropy.coordinates import SkyCoord
+        import astropy.units as u
+        sc = SkyCoord(1*u.deg, 2*u.deg)
+        output =' \n'.join(["sc."+attrname for attrname in dir(sc) if not attrname.startswith('_')])
+
+    except Exception as e:
+        output = str(e)
+
+    with open('../../docs/coordinates/TabCompleteList.txt', 'w') as rst_file:

```

Comment:
Assuming we want to do it this way, we'll have to also ask git to ignore this file.